ios/ZulipMobile.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		13B07FBC1A68108700A75B9A /* AppDelegate.m in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB01A68108700A75B9A /* AppDelegate.m */; };
+		13B07FBC1A68108700A75B9A /* AppDelegate.mm in Sources */ = {isa = PBXBuildFile; fileRef = 13B07FB01A68108700A75B9A /* AppDelegate.mm */; };
```

Comment:
Yeah. I didn't look closely at all of it, because it was clear from the commit message it wasn't intended except the parts that were actually about renaming that one source file. But I did notice that a lot of it looked like the stuff CocoaPods inserted for us.